docs/use_case/postv1.md

```diff
@@ -0,0 +1,61 @@
+This initial version of the demo #post app will allow you to create and visualise your own post(s). It will also showcase examples of the ONEM's python SDK usage.
+
+So let's activate our virtualenv and navigate to the **postv1** folder. Make sure that:
+
+ - the database is created (even if we will store the posts in-memory).
+ - local django server is started.
+ - the server is exposed publicly.
+ - the application is registered.
+All of the above steps are explained in the [run locally](/getting_started/python_run_local/) section of the docs.
+
+Now we can access the app from the test client. We should get the below menu:
+```bash
+#POSTV1 MENU
+Please choose a username
+--Send username
+```
+
+Send the desired username and we should be greeted with the landing menu:
+```bash
+#POSTV1
+A Add post
+B My posts
+--Reply A-B
+```
+
+You can go ahead and create a post or two which you can later visualise in the **My post** section of the app.
+
+Code-wise you can inspect the below classes from the *views.py* file:
+
+ - *class AddPostView(View)* - used for creating a new post and caching it in-memory. This class uses the SDK's **Form** structure. A **Form** contains **FormItemsContent** items which have specific properties like **name**, **description**, **header** etc:
+```
+FormItemContent(type=FormItemContentType.string,
```

Comment:
You need to update to latest onemsdk. This has been changed

Reply:
v1-v3 modified to match the new sdk version